src/__tests__/types/type.test.ts

```diff
@@ -0,0 +1,6 @@
+import fastify = require('fastify');
```

Comment:
why do you combine `import` and `require` here?

Reply:
ok, I'll change it.
thank you for your feedback.

Reply:
@PavelPolyakov Please review my code again.